src/book/02-system/01-foundations/02-build-system.mdx

```diff
@@ -132,6 +132,7 @@ As one might expect, `./b build` will build all enabled roles and required depen
 ```
 
 If unit tests are enabled in the CMake configuration (`-DBUILD_TESTS=ON`) then `./b build test` can be run to execute all of the unit tests.
+Test logs can be accessed by running `./b shell` and navigating to `/home/nubots/build/Testing/Temporary`.
```

Comment:
```suggestion
If unit tests are enabled in the CMake configuration (`-DBUILD_TESTS=ON`) then `./b build test` can be run to execute all of the unit tests. Test logs can be accessed by running `./b shell` and navigating to `/home/nubots/build/Testing/Temporary`.
```

Join lines since they're in the same paragraph. Could also separate with a new line if you wanted a new paragraph.